release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -177,6 +177,41 @@ The documentation for {olmv1}, which has been in Technology Preview starting in
 
 The updated location and guide name reflect a more focused documentation experience and aims to differentiate between {olmv1} and {olmv0}.
 
+[id="ocp-4-17-extensions-new-features_{context}"]
+==== {olmv1} Technology Preview features
+
+This release introduces the following new features to {olmv1} Technology Preview:
+
+Custom resource definition (CRD) upgrade safety::
+When you update a custom resource definition (CRD) that is provided by a cluster extension, {olmv1-first} now runs a CRD upgrade safety preflight check to ensure backwards compatibility with previous versions of that CRD. The CRD update must pass the validation checks before the change is allowed to progress on a cluster.
```

Comment:
Since these are both already spelled out before this sentence:
```suggestion
When you update a CRD that is provided by a cluster extension, {olmv1} now runs a CRD upgrade safety preflight check to ensure backwards compatibility with previous versions of that CRD. The CRD update must pass the validation checks before the change is allowed to progress on a cluster.
```